sdk/containerregistry/azure-containers-containerregistry/.run/ContainerRegistryClientAsyncTest (1).run.xml

```diff
@@ -0,0 +1,25 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
What are these files? Also do these contain leaked secrets that need to be rolled?

Reply:
Nothing of value here - they all contains old secrets but these are intellij run configuration files that made my life easier while debugging\testing. I will remove them. :)